http/swagger.yml

```diff
@@ -8880,6 +9019,18 @@ components:
           $ref: "#/components/schemas/Links"
     NotificationRuleBase:
       type: object
+      required:
+        - id
+        - notifyEndpointID
+        - orgid
+        - status
+        - name
+        - offset
+        - limitEvery
+        - limit
+        - tagRules
+        - description
```

Comment:
notifyEndpointID , description, offset, Limit, LimitEvery are not required.

Reply:
Thanks for catching these, will update.